src/components/LeaderboardCard/LeaderboardCard.tsx

```diff
@@ -0,0 +1,26 @@
+import "./LeaderboardCard.scss"
+import fallbackProfile from "../../../public/assets/images/default-profile-image.png"
```

Comment:
You don't have to give a relative path to the public folder, you generally do not import file from it.  Vite can figure out the path for you.

```tsx
          <img
          src={profileImage ?? "./assets/images/default-profile-image.png"}
          className="leaderboard-card__img-div leaderboard-card__img-div--img"
        />
```

Reply:
You will have to change how you use it in the Header as well.